contracts/base/PeripheryPayments.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC20Minimal} from "@uniswap/v4-core/contracts/interfaces/external/IERC20Minimal.sol";
+
+import {IPeripheryPayments} from "../interfaces/IPeripheryPayments.sol";
+
+import "../libraries/TransferHelper.sol";
+
+abstract contract PeripheryPayments is IPeripheryPayments {
+    /// @inheritdoc IPeripheryPayments
+    function sweepToken(address token, uint256 amountMinimum, address recipient) public payable override {
+        uint256 balanceToken = IERC20(token).balanceOf(address(this));
+        require(balanceToken >= amountMinimum, "Insufficient token");
+
+        if (balanceToken > 0) {
+            TransferHelper.safeTransfer(IERC20Minimal(token), recipient, balanceToken);
+        }
+    }
+
+    /// @inheritdoc IPeripheryPayments
+    function refundETH() external payable override {
```

Comment:
nit: why payable here?

Reply:
yea thats weird - we should look in v3 sdk to see if we ever use this with value. since it just sends back to msg.sender anyways it seems useless